discovery-infra/test_infra/tools/terraform_utils.py

```diff
@@ -40,8 +40,8 @@ def get_state(self) -> str:
         self.tf.read_state_file(self.STATE_FILE)
         return self.tf.tfstate
 
-    def set_new_vip(self, api_vip: str) -> None:
-        self.change_variables(variables={"api_vip": api_vip}, refresh=True)
+    def set_new_vip(self, api_vip: str, ingress_vip: str) -> None:
```

Comment:
maybe we should have function that will set list of dns_names to ip? 
i mean we will not add those values to tf in advance.
Sort of dynamic tf update?

Reply:
Actually, I already introduced dns_records field on https://github.com/openshift/assisted-test-infra/pull/791
This is currently not used. I guess we should take advantage of it.

Reply:
I opened a ticket since it's a bigger task
https://issues.redhat.com/browse/MGMT-6928